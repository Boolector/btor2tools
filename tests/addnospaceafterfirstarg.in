1 zero 32
2 add 32 1nospace
