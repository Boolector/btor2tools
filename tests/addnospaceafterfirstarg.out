*** catbtor: parse error in 'addnospaceafterfirstarg.in' line 2: expected space after first argument
